### DISCUSSION TITLE
When will GTD be available

### DISCUSSION THREAD

**Discussion by my0811** [nvim-neorg/neorg#1540] [General]
When will GTD be available

**Comment by GitMurf** [comment]
I am wondering as well. I haven't been able to find many informational updates recently on it so curious if it is still a priority and what the status is.

**Reply by max397574** [reply]
it is a priority but things which are required for it are being worked on rn

**Reply by liubog2008** [reply]
Where can I find updates on the progress of the GDT? Is there any preview version?

**Reply by austinbutler** [reply]
There's this pinned issue but no recent updates: https://github.com/nvim-neorg/neorg/issues/695
